scripts/Get-VersionConstants.ps1

```diff
@@ -1,8 +1,17 @@
 <#
 .SYNOPSIS
 Extract the version number from build.props.
+
+.PARAMETER Previous
+Extract the previous version number rather than the current version number.
```

Comment:
>number [](start = 68, length = 6)

When we create packages, we use the current version number; when we hide them, we hide the packages with the previous version number. So we add this switch to the `Get-VersionConstants` command.